src/video/x11/SDL_x11window.c

```diff
@@ -1888,6 +1871,39 @@ X11_FlashWindow(_THIS, SDL_Window * window, SDL_FlashOperation operation)
     return 0;
 }
 
+bool SDL_X11_SetWindowTitle(Display* display, Window xwindow, char* title) {
+    Atom _NET_WM_NAME = X11_XInternAtom(display, "_NET_WM_NAME", False);
+    XTextProperty titleprop;
+    int conv = X11_XmbTextListToTextProperty(display, (char**) &title, 1, XStdICCTextStyle, &titleprop);
+    Status status;
+
+    if (conv == 0) {
+        X11_XSetTextProperty(display, xwindow, &titleprop, XA_WM_NAME);
+        X11_XFree(titleprop.value);
+    // we know this can't be a locale error as we checked X locale validity in X11_VideoInit
+    } else if (conv <= 0) {
+        SDL_LogError(SDL_LOG_CATEGORY_VIDEO, "X11 reporting it's out of memory");
+        return EXIT_FAILURE;
+    } else { // conv >= 0
+        SDL_LogWarn(SDL_LOG_CATEGORY_VIDEO, "%d characters were not convertable to the current locale!", conv);
+        return EXIT_FAILURE;
+    }
+
+#ifdef X_HAVE_UTF8_STRING
+    status = X11_Xutf8TextListToTextProperty(display, (char **) &title, 1, XUTF8StringStyle, &titleprop);
+    if (status == Success) {
+        X11_XSetTextProperty(display, xwindow, &titleprop, _NET_WM_NAME);
+        X11_XFree(titleprop.value);
+    } else {
+        SDL_LogError(SDL_LOG_CATEGORY_VIDEO, "Failed to convert title to UTF8! Bad encoding, or bad Xorg encoding? Window title: «%s»", title);
+        return EXIT_FAILURE;
+    }
+#endif
+
+    X11_XFlush(display);
+    return EXIT_SUCCESS;
```

Comment:
EXIT_SUCCESS and EXIT_FAILURE may not be defined. Use 0 for EXIT_SUCCESS.

Reply:
... and `-1` for EXIT_FAILURE

Reply:
Sorry about that, I thought these symbols were standard. Fixed in 513e0eb.